app/api/stage-configuration.js

```diff
@@ -0,0 +1,20 @@
+const Wreck = require('@hapi/wreck')
+const { applicationApiUri } = require('../config')
+
+async function getAllStageConfigurations () {
+  const url = `${applicationApiUri}/stageconfiguration`
+  try {
+    const response = await Wreck.get(url, { json: true })
+    if (response.res.statusCode !== 200) {
+      return null
+    }
+    return response.payload
+  } catch (err) {
+    console.log(err)
+    return null
```

Comment:
can we return an empty array here so later we wouldn't have to remember to handle null separately